bitnami/grafana/templates/deployment.yaml

```diff
@@ -0,0 +1,167 @@
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        helm.sh/chart: {{ include "grafana.chart" . }}
+{{- if .Values.podAnnotations }}
+      annotations:
+         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+         checksum/dashboard-provider: {{ include (print $.Template.BasePath "/dashboard-provider.yaml") . | sha256sum }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+    spec:
+      {{- if .Values.schedulerName }}
+      schedulerName: {{ .Values.schedulerName | quote }}
+      {{- end }}
+{{- include "grafana.imagePullSecrets" . | indent 6 }}
+     {{- if .Values.securityContext.enabled }}
+      securityContext:
+        runAsUser: {{ .Values.securityContext.runAsUser }}
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        runAsNonRoot: {{ .Values.securityContext.runAsNonRoot }}
+      {{- end }}
+{{- with .Values.nodeSelector }}
+      nodeSelector:
+{{-  tpl (toYaml .) $ | indent 8 }}
+{{- end }}
+{{- with .Values.affinity }}
+      affinity:
+{{-  tpl (toYaml .) $ | indent 8 }}
+{{- end }}
+{{- with .Values.tolerations }}
+      tolerations:
+{{- tpl (toYaml .) $ | indent 8 }}
+{{- end }}
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        envFrom:
+        - configMapRef:
+            name: "{{ include "grafana.fullname" . }}-envvars"
+        - secretRef:
+            name: "{{ include "grafana.fullname" . }}-secret"
+        {{- if .Values.smtp.existingSecret }}
+        - name: GF_SMTP_USER
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.smtp.existingSecret }}
+              key: {{ .Values.smtp.existingSecretUserKey | default "user" }}
+        - name: GF_SMTP_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.smtp.existingSecret }}
+              key: {{ .Values.smtp.existingSecretPasswordKey | default "password" }}
+        {{- end }}
+{{- if .Values.extraEnvVars }}
+{{ toYaml .Values.extraEnvVars | indent 8 }}
+{{- end }}
+        volumeMounts:
+        {{- if .Values.config.useGrafanaIniFile }}
+        - name: grafana-ini
+          mountPath: /opt/bitnami/grafana/conf/grafana.ini
+          subPath: grafana.ini
+        {{- end }}
+        {{- if .Values.config.useCustomIniFile }}
+        - name: custom-ini
+          mountPath: /opt/bitnami/grafana/conf/custom.ini
+          subPath: custom.ini
+        {{- end }}
+        - name: data
+          mountPath: /opt/bitnami/grafana/data
+        {{- if .Values.dashboardsProvider.enabled }}
+        - name: dashboards-provider
+          mountPath: /opt/bitnami/grafana/conf/provisioning/dashboards
+        {{- end }}
+        {{- range .Values.dashboardsConfigMaps }}
+        - name: {{ . }}
+          mountPath: /opt/bitnami/grafana/dashboards
+        {{- end }}
```

Comment:
Does this range works? Can you mount more than one **configmap** in the same **mountpath**?

Reply:
I changed the approach to include the filename so we can mount more than one in a single directory. Thanks for realize about this!!